app/controllers/lookbook/application_controller.rb

```diff
@@ -1,5 +1,7 @@
 module Lookbook
   class ApplicationController < ActionController::Base
+    before_action :base_auth, if: config.base_auth && Rails.env.production?
```

Comment:
Just looking at this, I wonder if checking for `Rails.env.production?` is needed here - it feels like the end user could do that when setting the `config.base_auth` setting value if that is the behaviour they wanted. For example:

```rb
# config/application.rb
config.lookbook.base_auth = Rails.env.production?
```

So Lookbook itself is not making any additional assumptions past the config value check. What do you think?